torch/nested/_internal/ops.py

```diff
@@ -628,7 +629,7 @@ def copy_default(func, *args, **kwargs):
         raise RuntimeError(
             "copy_ only supports Nested Tensors that have same size and the exact same offset tensor."
         )
-    inp.values().copy_(src.values())
+    inp._values.copy_(src._values)
```

Comment:
Can you add a comment here on why it's actually important for us to use `_values` instead of `values()` here